### PR TITLE
(maint) Release 0.18.1

### PR DESCRIPTION
**Pull request by jpogran**
This commit updates the changelog and package.json for the 0.18.1
release.

### CODE REVIEW

**Comment by jpogran** [comment]
Pending #519 and #513

**Comment by glennsarti** [comment]
... and https://github.com/lingua-pupuli/puppet-vscode/pull/522

**Comment by glennsarti** [comment]
522 is merged.  Unblocking

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,31 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+## [0.18.1] - 2019-05-31
```
Comment (glennsarti) [inline CHANGELOG.md]: nit: wrong date

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,31 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
...
+
+### Added
+
+- ([GH-505](https://github.com/lingua-pupuli/puppet-vscode/issues/505)) Add hide `PDK New Module` titlebar button
+- ([GH-507](https://github.com/lingua-pupuli/puppet-vscode/issues/507))([GH-508](https://github.com/lingua-pupuli/puppet-vscode/issues/508)) Document Puppet Version Support
```
Comment (glennsarti) [inline CHANGELOG.md]: Missing 522

Comment (glennsarti) [reply CHANGELOG.md]: ~~i.e. updates to editor services and editor syntax~~

OH I see, you added those explicitly into the Fixed section

**Comment by jpogran** [comment]
I have to commit package-lock.json, it tracks version too. If i dont it is still on old version.

File: CHANGELOG.md (hunk 1 above)
Comment (jpogran) [reply CHANGELOG.md]: :facepalm:

**Comment by glennsarti** [comment]
💩 to package-lock :-(

**Pull request merged**